packages/manager/src/managers/git/GitManager.ts

```diff
@@ -325,54 +313,6 @@ export class GitManager extends BaseManager {
 		}
 	}
 
-	async checkHasWriteAPIToken(
```

Comment:
I think we can leave the method in the manager and SM API since we may want to separate token management from the Git integration.

However, if you think it's better to remove the focus the APIs on the current feature implementation, we can make that change. In that case, it's probably best to remove all of the Write API token endpoints + methods and update `linkRepo` to require the token as a parameter.